service/matching/forwarder.go

```diff
@@ -91,19 +93,21 @@ func newForwarder(
 	taskListID *taskListID,
 	kind types.TaskListKind,
 	client matching.Client,
+	isolationGroups []string,
```

Comment:
Is this intended to be that private-package type?

Reply:
it's allisolationgroups if isolation is enabled, it's nil if not